src/ServiceControl.Plugin.Nsb6.SagaAudit/Messages/SagaChangeOutput.cs

```diff
@@ -1,15 +1,24 @@
 namespace ServiceControl.EndpointPlugin.Messages.SagaState
 {
     using System;
+    using System.Runtime.Serialization;
 
+    [DataContract]
     class SagaChangeOutput
     {
+        [DataMember]
         public string MessageType { get; set; }
+        [DataMember]
         public DateTime TimeSent { get; set; }
+        [DataMember]
         public DateTime? DeliveryAt { get; set; }
+        [DataMember]
         public TimeSpan? DeliveryDelay { get; set; }
```

Comment:
The DataContractSerializer seems to have a major issue with round-tripping TimeSpan values. http://stackoverflow.com/questions/27925632/facing-issue-with-timespan-in-rest-service-serialization for example